README.md

```diff
@@ -560,9 +615,147 @@ module.exports = buildWebpackConfig([
+        <meta name="twitter:description" content="Home description" />
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta name="twitter:site" content="Client Twitter handle" />
+
+        {/* Chrome for Android */}
+        <meta name="theme-color" content="Hex value" />
+      </Helmet>
+
+      <main className="homePage">
+        {/* Homepage content */}
+      </main>
+    )
+  }
+}
+
+```
+
+Please note that some of these meta can be put directly in `src/index.html`, as they are probably the same for all pages. Specifically, `robots`, `copyright`, `og:type`, `twitter:card`, `twitter:site`.
+
+## Add `react-snap` in `src/index.js`
+
+```
+import {render, hydrate} from 'react-dom'
+
+renderApp()
+
+function renderApp() {
+  if (rootElement.hasChildNodes()) {
+    hydrate(<App />, rootElement)
+  } else {
+    render(<App />, rootElement)
+  }
+}
+```
+
+## Add `react-snap` to `package.json`
+
+```
+"scripts": {
+	...
+  "postbuild": "react-snap"
+},
+"reactSnap": {
+  "puppeteerArgs": [
+    "--no-sandbox"
+  ]
+}
+```
+
+Note: the puppeteerArgs avoid the build to break on the Bitbucket pipelines.
+
+## Add the `react-snap` config in `bitbucket-pipelines.yml`
+
+Add it in `script`, right before `git clone --branch="master"` ...
+
+```
+- apt-get update; apt-get install -y gettext-base;
+- echo 'deb http://dl.google.com/linux/chrome/deb/ stable main' > /etc/apt/sources.list.d/chrome.list
+- wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
+- set -x && apt-get update && apt-get install -y xvfb google-chrome-stable
+- wget -q -O /usr/bin/xvfb-chrome https://bitbucket.org/atlassian/docker-node-chrome-firefox/raw/ff180e2f16ea8639d4ca4a3abb0017ee23c2836c/scripts/xvfb-chrome
+- ln -sf /usr/bin/xvfb-chrome /usr/bin/google-chrome
+- chmod 755 /usr/bin/google-chrome
+```
+
+## OK, setup done! Now, how do I check if it is working?
+
+Run `yarn build`. After the build is complete, you will see some folders with an `index.html` file in them. Also `react-snap` shows its progress in the terminal right after `yarn build` task is complete.
+
+## Basic troubleshooting: `react-snap` works properly, but no links are found
+
+You probably forgot to add `<a href="{dynamicLink}">` to the page. React-snap renders all pages looking for `<a>` tags. It then follows the `href` to render the subsequent pages. If no `<a>` tags are found, then no links are crawled.
+
+This is particularly important, as some routers hide their logic in an `onClick` event handler, and don't compile your links to actual `<a>` tags by default (e.g. `mobx-state-router`).
+
+## Basic troubleshooting: I get a weird error for 404 pages
+
+On 404 pages, `react-snap` requires you to have the string `404` to be part of the `<title>`, such as
+
+```
+<title>404 - Page not found</title>
+```
+
+or the following meta to be added to the `<Helmet>` component:
+
+```
+<meta name="prerender-status-code" content="404" />
```

Comment:
this is not implemented

Reply:
Good catch, I'll reference [#91](https://github.com/stereobooster/react-snap/issues/91) instead